### PR TITLE
Updated the documentation markdown files and improved documentation i…

### PR DESCRIPTION
**Pull request by armstrongmsg**
…n the code files

### CODE REVIEW

File: doc/proportional-derivative-controller.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Controller plugin - "proportional-derivative"
+
+## Start Scaling
+
+### Expected parameters
+
+#### check_interval
+
+Time between application state checks, in seconds. The controller checks the application state every check_interval seconds.
```
Comment (igornatanael) [inline doc/proportional-derivative-controller.md:9]: Is this comma really needed?

File: README.md [hunk 2]
```diff
@@ -1,80 +1,164 @@
...
+# Deploy
+
+To install, configure and run the Controller component you will need a virtual machine with the configurations described below.
+
+**Minimal configuration**
+```
+OS: Ubuntu 14.04
+CPU: 1 core
+Memory: 2GB of RAM
+Disk: there is no disk requirements.
```
Comment (igornatanael) [inline README.md]: Please add some requirement for disk.

File: doc/proportional-derivative-controller.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+The minimum possible value of resources that can be allocated to the application. Must be in range [0,100].
+
+#### max_cap
+
+The maximum possible value of resources that can be allocated to the application. Must be in range [0,100].
+
+#### metric_rounding
+
+The amount of decimal digits considered when taking scaling decisions.
+e.g.: If the scaler gets a metric value 0.567 from the metric source and metric_rounding = 2 then the value the scaler will use is 0.57.
```
Comment (igornatanael) [inline doc/proportional-derivative-controller.md:30]: The line is not jumping here.

**Review by igornatanael — CHANGES_REQUESTED** [review]
Please, consider these few suggests

File: README.md (hunk 2 above)
Comment (armstrongmsg) [reply README.md]: Added "Disk: 20GB"

**Pull request merged**